library/core/src/mem/maybe_uninit.rs

```diff
@@ -726,16 +729,16 @@ impl<T> MaybeUninit<T> {
         }
     }
 
-    /// Drops the contained value in place.
```

Comment:
Here is another example of a magic phrase: "drop in place". Does that mean to drop it without moving it first, or is it equivalent to just "drop". When does the difference (if there is one) matter? I honestly don't know the answer, which I why I did not yet propose a change for that part, but it could use some clarification.

Reply:
Maybe it should refer to that function then...

Reply:
Yeah, maybe it should.

Reply:
Done in the other PR.